app/Http/Controllers/JwtClaimsSupplier.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Http\Resources\JwtClaims;
+use App\Models\User;
+
+class JwtClaimsSupplier extends Controller
+{
+    /**
+     * Return custom claims (for inclusion in JWT) for the given user
+     */
+    public function __invoke(User $user): JwtClaims
```

Comment:
I wouldn't use the magic Laravel Model injection. I'd rather explicitly find the exact param from query using $request->get(...) and query it User::find($id)

Reply:
Thank you for the suggestion. I trust your experience on this. Changed it in latest revision.